src/components/PlaybookProgress.js

```diff
@@ -552,17 +563,16 @@ class PlaybookProgress extends Component {
       this.updateGlobalPlaybookStatus(playbook.name, 0, STATUS.IN_PROGRESS);
       this.props.updatePageStatus(STATUS.IN_PROGRESS);
 
-      playbook.action(this.logMessage)
+      let promise = playbook.action(this.logMessage);
+      if(!(promise instanceof Promise)) {
+        // this step gave us back something that was not a Promise.
+        // we will assume that is setp is finished and move on to the next one.
```

Comment:
should this read "we will assume that this step is finished and move on to the next one." ?